main.go

```diff
@@ -82,6 +83,7 @@ func main() {
 	flag.BoolVar(&watchAllNamespaces, "watch-all-namespaces", true,
 		"Watch for custom resources in all namespaces, if set to false it will only watch the runtime namespace.")
 	flag.DurationVar(&rateLimitInterval, "rate-limit-interval", 5*time.Minute, "Interval in which rate limit has effect.")
+	flag.BoolVar(&insecureAllowHTTP, "insecure-allow-http", false, "Enable the use of HTTP Scheme (no HTTPS) across all controller level objects. This is not recommended for production environments")
```

Comment:
The default must be `true` according to [RFC 4](https://github.com/fluxcd/flux2/tree/c7e158aaa7b4c250f6c5d9fb1518a3b8bff7db36/rfcs/0004-insecure-http).

Reply:
Done